test/templates/api/repositories/template-list-filter/004-list-ok-wrong-id/expected-response.json

```diff
@@ -0,0 +1,115 @@
+{
+  "repository": {
```

Comment:
How can wrong ID in `filter` return all components? I would say that it returns empty slice.

Reply:
[905b218](https://github.com/keboola/keboola-as-code/pull/2144/commits/905b218b1fa9bfaedd62b73ca09946de26845a6d)